library/core/src/iter/traits/iterator.rs

```diff
@@ -2434,6 +2434,7 @@ pub trait Iterator {
     /// assert!(result.is_err());
     /// ```
     #[inline]
+    #[cfg_attr(not(bootstrap), allow(duplicate_bounds))]
```

Comment:
Why is the lint reported here?

Reply:
Because `Try` and `TryV2` are the same thing now. cc @scottmcm

Reply:
Oh, good point, I never cleaned that alias up after the bootstrap compiler updated.  New PR: https://github.com/rust-lang/rust/pull/88223